db/migrate/20200908223206_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments do |t|
+      t.string :comment_title
+      t.text :comment_body
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    # remove_column :comments, :author_id
+     remove_foreign_key :comments, column: :author_id
```

Comment:
- [ ] This line(11) is giving the following error when I try to run your migrations because is trying to remove a column that doesn't exist yet let's fix it (this column should exist by the way so it shouldn't be removed)
![image](https://user-images.githubusercontent.com/52209067/92649192-e1349000-f2a7-11ea-9673-7a9635638c4e.png)

Reply:
We forgot to remove this line after correcting our table. Issue corrected!